@types/@now_build-utils/index.d.ts

```diff
@@ -0,0 +1,107 @@
+// https://zeit.co/docs/v2/deployments/builders/developer-guide/
+
+declare module '@now/build-utils/lambda' {
+  type Lambda = BuildUtils.Lambda;
+  export default Lambda;
+}
+
+declare module '@now/build-utils' {
```

Comment:
You can probably remove these now

Reply:
Done 810b951f83b3ac9e3ab425a1f4461b1cc9d414d6